translations/strings.yaml

```diff
@@ -82,3 +98,172 @@ splittunnel:
+  infoText: For any questions about setting app permissions, we've provided some helpful tips on our website.
+  infoLinkText: Learn more
+  
+#adding multi-hop feature
+multiHopFeature:
+  #Screen MacOS/134
+  singleToggleHeader: Single-hop
+  multiToggleHeader: Multi-hop
+  #Screen MacOS/140
+  cardHeader: What is multi-hop VPN?
+  cardBody: Multi-hop VPN routes your traffic through two servers instead of one for extra security and privacy. This may slow down your connection.
+  entryLocationHeader: Select entry location
+  exitLocationHeader: Select exit location
+  infoText: Exit location represents your main VPN server.
+  #Screen MacOS/136
+  pageHeader: Select entry location
+  #Screen MacOS/145
+  pageHeader: Select exit location
+  #Screen MacOS/142
+  connectionsHeader: Recent connections
+  arrowAltText: To
+  #Screen MacOS/138
+  searchResultsAlert: No results. Try a different search
+
+customDNSSettings:
+  #Screen 830
+  infoAlert: VPN must be off to change these settings
+  dnsNavItem: DNS settings
+  #Screen 832
+  defaultToggle: Default
+  defaultRadioHeader: Use default DNS
+  defaultRadioBody: Automatically use Mozilla VPN-protected DNS
+  #Screen 837
+  advancedToggle: Advanced
+  adblockRadioHeader: Use adblock DNS
+  adblockRadioBody: Block internet ads with VPN
+  antitrackRadioHeader: Use anti-tracking DNS
+  antitrackRadioBody: Block harmful domains with VPN
+  adblockAntiTrackRadioHeader: Use adblock and anti-tracking DNS
+  adblockAntiTrackRadioBody: Block internet ads and harmful domains with VPN
+  customDNSRadioHeader: Use custom DNS
+  customDNSRadioBody: Resolve websites using a custom DNS
+  dnsFieldPlaceHolder: Enter custom DNS
+  #Screen 839
+  customDNSAlert: Invalid DNS, please try a new one
+
+  #support workflow feature in 2.5. Signed-in spec the same as not signed in but with email field removed.
+  SupportWorkflow:
+    #screen MacOS / 29
+    navLabel: Contact support
+    #screen MacOS / 30
+    pageHeader: Contact support
+    fieldCategoryHeader: Enter your email
+    emailFieldPlaceholder: Email address
+    confirmEmailPlaceholder: Confirm email address
+    fieldCategoryHeader: How can we help you with Mozilla VPN?
+    dropdownLabel: Choose a category
+    subjectFieldPlaceholder: Subject (optional)
+    issueFieldPlaceholder: Describe issue…
+    disclaimerText: When you submit, Mozilla VPN will collect technical and interaction data with your email to help our support team understand your issue.
+    privacyNoticeLinkText: Mozilla VPN Privacy Notice
+    primaryButtonText: Submit
+    secondaryActionText: Cancel
+    #screen MacOS / 31
+    dropdownListItem: Payment and billing
+    dropdownListItem: Account issues
+    dropdownListItem: Technical issues
+    dropdownListItem: Request features
+    dropdownListItem: Other
+    #screen MacOS 32
+    responseHeader: Thank you!
+    responseBody: Our support team will contact you at _email_ once we've been able to look at your issue.
```

Comment:
_email_ is not the format used for variables. It should be `%1`, and with a comment explaining what replaces the variable.